client/src/components/timeline-window-component/clientWindow.jsx

```diff
@@ -0,0 +1,183 @@
+      <Box paddingTop={2} paddingBottom={2} paddingLeft={2}>
+        <Grid container direction="row" alignItems="center">
+          <Grid item xs={11}>
+            <Typography variant="h5" align="left">Activity Timeline</Typography>
+          </Grid>
+          <Grid item xs={1}>
+          <IconButton
+        aria-label="more"
+        id="long-button"
+        aria-controls={open ? 'long-menu' : undefined}
+        aria-expanded={open ? 'true' : undefined}
+        aria-haspopup="true"
+        onClick={handleClickDots}
+      >
+        <MoreVertIcon />
+      </IconButton>
+      <Menu
+              anchorEl={anchorElDots}
+              open={openDots}
+              onClose={handleCloseDots}
+              MenuListProps={{
+                "aria-labelledby": "split-button",
+              }}
+              >
+                <MenuItem onClick={handleClickDots} sx={{ justifyContent: "center" }} >
+                  <DownloadIcon sx={{color: "grey", paddingRight: "10px"}}/>
+                  Export as JSON
+                </MenuItem>
+                </Menu>
+          </Grid>
+        </Grid>
+      </Box>
+
+          <CardContent>
+          
+            <ButtonGroup
+              variant="contained"
+              style={{ width: "100%", borderRadius: "8px" }}
+              aria-label="split button"
+              ref={buttonGroupRef}
+              color="primary"
+              sx={{'& .MuiButtonBase-root': {backgroundColor: "#3568E5"}}}
+              
+            >
+              <Button fullWidth startIcon={<AddIcon />}>Add New Entry</Button>
+              <Button onClick={handleClick}>
+                <ArrowDropDownIcon />
+              </Button>
+              <Menu
+              anchorEl={anchorEl}
+              open={open}
+              onClose={handleClose}
+              MenuListProps={{
+                "aria-labelledby": "split-button",
+              }}
+              PaperProps={{
+                style: {
+                  width: buttonGroupRef
+                    ? buttonGroupRef.current?.clientWidth
+                    : null,
+                },
+              }}
+              >
+                <MenuItem onClick={handleClose} sx={{ justifyContent: "center" }} >
+                  Placed Job Seeker
+                </MenuItem>
+                <MenuItem onClick={handleClose} sx={{ justifyContent: "center" }} >
+                  Other Note
+                </MenuItem>
+
+              </Menu>
+            </ButtonGroup>
+
+           
+           
+            <TabContext value={value}>
+              <TabList onChange={handleChange} sx={{
+        '& .MuiTabs-flexContainer': { display: 'flex', width: "100%", gap: -2 }, 
+        '& .MuiButtonBase-root': {padding: 0}
+      }}>
+                <Tab label="ALL" value="1" />
+                <Tab label="CONTACTED" value="2" />
+                <Tab label="UPDATES" value="3" />
+                <Tab label="NOTES" value="4" />
```

Comment:
The spacing between different tabs here looks a bit uneven
<img width="495" alt="image" src="https://github.com/uoftblueprint/tces/assets/58835213/f04b3bb1-62f3-4205-ba8c-a8acd09aa60d">